contracts/components/scanners/ScannerRegistryEnable.sol

```diff
@@ -132,7 +132,7 @@ abstract contract ScannerRegistryEnable is ScannerRegistryManaged {
      * @param value true if enabling, false if disabling.
      */
     function _afterScannerEnable(uint256 scannerId, Permission permission, bool value) internal virtual {
```

Comment:
Do we plan to remove these hooks in a future PR?

Reply:
I thought about it several times, never came up. I'm doing the new `NodeRunnerRegistry` without them, so I'm all up for removing them from `AgentRegistry` too. I'll do a separate PR.